cms/static/sass/views/_outline.scss

```diff
@@ -754,3 +754,8 @@
   }
 
 }
+
+.ora1-components-list {
```

Comment:
we should add this styling to what exists already in _system-feedback.scss, or use a class from that file that is appropriate. If this is special cased we can add a // CASE: deprecation warning styling element. I would also use a class name that labels this as a deprecation warning instead of stipulating ora1 specifically.